### ISSUE TITLE
Is it possible to create a DEAD contract?

### ISSUE THREAD

**Issue by hacker-DOM** [ethereum/yellowpaper#831]
The Yellow Paper contains the following logic under contract creation:

<img width="438" alt="Screenshot 2021-12-06 at 7 38 15 PM" src="https://user-images.githubusercontent.com/18601956/144911013-07ba92a8-f144-4314-bc13-a106c73ffcc9.png">

`DEAD(s, a)` is defined as:

<img width="519" alt="Screenshot 2021-12-06 at 7 39 34 PM" src="https://user-images.githubusercontent.com/18601956/144911202-c23230ae-b037-46f5-9130-5a3e101efe82.png">

Notably, the `nonce` must be 0.

In the first figure, `a` represents the address of the created account. However:

<img width="517" alt="Screenshot 2021-12-06 at 7 41 02 PM" src="https://user-images.githubusercontent.com/18601956/144911421-85356172-3cb5-42dd-9d4d-80ec7741bbf4.png">

The nonce of the created account is initially set to 1.

TLDR: Am I correct in saying that the case in Figure 1 cannot occur, and hence is redundant?

**Comment by yperbasis** [comment]
Yeah, I think that the second condition in (102) is redundant.
